lib/tasks/workflow_update.rake

```diff
@@ -0,0 +1,66 @@
+require 'csv'
+
+namespace :workflow do
+  desc 'Update workflows'
+  task update: :environment do
+    # excluded_accounts = ["qa-2.commons-archive.org", "qa-3.commons-archive.org", "qa-12.commons-archive.org", "search-test-nd.commons-archive.org", "make-tenant.commons-archive.org", "delete-me.commons-archive.org", "delete-me-2.commons-archive.org", "nic4.commons-archive.org", "nic5.commons-archive.org", "delete-me-4.commons-archive.org", "catmango.commons-archive.org", "delete-me-when-done.commons-archive.org", "search.commons-archive.org"]
```

Comment:
```suggestion
```

Reply:
why exclude these accounts on staging? shouldn't these existing accounts, and future ones, be configured to the expected settings too?

Reply:
These specific accounts dont have proper fcrepo/solr endpoints and as such do not have an admin_set created, which errored and errored, as I added each errored tenant to the excluded_accounts. 

I ran this on staging already and this was the workflow I had come up with to address all the cases that the script would error lol. I ran a similar script on prod without the actions and only adding the info to the csv file I created when I ran it, I have that linked in the pr as well :)